src/readbin/ida.mli

```diff
@@ -9,6 +9,8 @@
 open Bap.Std
 open Core_kernel.Std
 
+exception Cmderr of string
```

Comment:
rename to `External_command_fail` and add docs, explaining what this exception should mean